### PR TITLE
change password updated

### PR DESCRIPTION
**Pull request by N00Bmaster2810**
## Proposed changes
Changed as per changes required in PR #94 
## Other information
API testing collection link: https://www.getpostman.com/collections/aba0b43b7e2e18f7d9a7

### CODE REVIEW

**Comment by N00Bmaster2810** [comment]
@jackfrost13 please have a look.

File: backend/app/routes/admin/changePassword.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    const error = new ErrorHandler(constants.ERRORS.DATABASE, {
+      statusCode: 500,
+      message: 'Mongo Error: Insertion Failed',
+      errStack: err,
+      user: userRecord.email,
+    });
+    return next(error);
+  }
+
+  //   eslint-disable-next-line no-underscore-dangle
```
Comment (jackfrost13) [inline backend/app/routes/admin/changePassword.js:58]: remove this comment. not required

Comment (N00Bmaster2810) [reply backend/app/routes/admin/changePassword.js]: done @jackfrost13

**Comment by jackfrost13** [comment]
closes #94

**Pull request merged**